WordPress/src/main/res/layout/fragment_my_site.xml

```diff
@@ -0,0 +1,309 @@
+                android:layout_below="@id/my_site_blavatar"
+                android:background="@color/my_site_button_bg">
+
+                <org.wordpress.android.widgets.WPTextView
+                    android:id="@+id/switch_site"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginLeft="@dimen/margin_large"
+                    android:background="?android:attr/selectableItemBackground"
+                    android:drawableLeft="@drawable/switch_site_button_icon"
+                    android:drawablePadding="@dimen/margin_small"
+                    android:gravity="center_vertical"
+                    android:minHeight="0dp"
+                    android:paddingBottom="@dimen/my_site_switch_site_button_padding_bottom"
+                    android:paddingTop="@dimen/my_site_switch_site_button_padding_top"
+                    android:text="@string/switch_site_button"
+                    android:textColor="@color/my_site_dark_blue"
+                    android:textSize="@dimen/text_sz_medium" />
+
+            </LinearLayout>
+
+        </RelativeLayout>
+
+        <!--View Site-->
+        <RelativeLayout
+            style="@style/MySiteListRowLayout"
+            android:layout_marginTop="@dimen/my_site_list_margin_top">
+
+            <ImageView
+                android:id="@+id/my_site_view_site_icon"
+                style="@style/MySiteListRowIcon"
+                android:contentDescription="@string/view_site"
+                android:src="@drawable/my_site_icon_view_site" />
+
+            <org.wordpress.android.widgets.WPTextView
+                android:id="@+id/my_site_view_site_text_view"
+                style="@style/MySiteListRowTextView"
+                android:layout_toRightOf="@id/my_site_view_site_icon"
+                android:text="@string/view_site" />
+
+        </RelativeLayout>
+
+        <!--Stats-->
+        <RelativeLayout style="@style/MySiteListRowLayout">
+
+            <ImageView
+                android:id="@+id/my_site_stats_icon"
+                style="@style/MySiteListRowIcon"
+                android:contentDescription="@string/stats"
+                android:src="@drawable/my_site_icon_stats" />
+
+            <org.wordpress.android.widgets.WPTextView
+                android:id="@+id/my_site_stats_text_view"
+                style="@style/MySiteListRowTextView"
+                android:layout_toRightOf="@id/my_site_stats_icon"
+                android:text="@string/stats" />
+
+        </RelativeLayout>
+
+        <!--PUBLISH-->
+        <LinearLayout style="@style/MySiteListHeaderLayout">
+
+            <org.wordpress.android.widgets.WPTextView
+                style="@style/MySiteListHeaderTextView"
+                android:text="@string/publish" />
+
+            <View style="@style/MySiteListSectionDividerView" />
+
+        </LinearLayout>
+
+        <!--Blog Posts-->
```

Comment:
None of the "buttons" in this layout give any visual feedback when pressed. Using `android:background="?android:attr/selectableItemBackground"` should do the trick, but you may need to change things around a bit so the selection state is applied to both the icon and the textView.